### DISCUSSION TITLE
Fetching AJAX content: swup with Fancyapp v4 modal

### DISCUSSION THREAD

**Discussion by DennysDionigi** [swup/swup#427] [Help]
Hi all, seems that using the "standard" fancyapp ajax api to fetch an element from a different page, it loads the entire page without any content at all.
Which is the right way to fetch ajax content using swup? (Maybe.. let's say inside a modal view || appending somewhere the data fetched?)

I'm on page **"FOO"**, need to **fetch()** page **"BAR#element1"**.

Let's suppose the website is _github.com_.

Thanks.

**Comment by hirasso** [comment]
@DennysDionigi could you create a reduced test case online somewhere, for example on [glitch.com](https://glitch.com)?

Otherwise we can't help here, since there are too many unknown factors in your question.
